source/docs/create_project.md

```diff
@@ -9,6 +9,17 @@ As discussed in our [quickstart guide](quick-start.html), the fastest way to get
 
 <pre><code class="shell">$ embark demo</code></pre>
 
+## Creating demo apps from templates
+
+By default, `embark demo` will create a demo application that is built with React. If we wish to create a demo application that's built with a different framework, we can use the `--template` option and either specify a supported template name or Git host URL.
+
+The following command creates a demo application built with the VueJS library:
+
+<pre><code class="shell">$ embark demo --template vue
+</code></pre>
+
+If the given template is a name (as opposed to a full Git URL), Embark will first look for a repository called `embark-[template_name]-template`, clone it, and create an application from the template's `demo` branch. This is very important: If we'd like to create a demo application from any Git repository, there has to be a dedicated `demo` branch.
```

Comment:
should you also specify the org too:
`embark-[template_name]-template` => `embark-framework/embark-[template_name]-template`